README.md

```diff
@@ -90,6 +90,15 @@ import (
 )
 
 func main() {
+  // Create signals channel to run server until interrupted
+  sigs := make(chan os.Signal, 1)
+  done := make(chan bool, 1)
+  signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+  go func() {
+    <-sigs
+    done <- true
+  }()
+
   // Create the new MQTT Server.
   server := mqtt.New(nil)
```

Comment:
To get this to work I had to change the import to be ```mqtt "github.com/mochi-mqtt/server/v2"```

Reply:
Very good catch! I had to do the same thing but I forgot to copy over imports.